tutorial/tutorial.thrift

```diff
@@ -113,7 +113,7 @@ struct Work {
  * Structs can also be exceptions, if they are nasty.
  */
 exception InvalidOperation {
-  1: i32 what,
+  1: i32 what_op,
```

Comment:
1. this naming convention is inconsistent with rest of tutorial.thrift file (camelCase)
2. changing it here requires update in all tutorials, not only C++